presto-product-tests/src/main/resources/sql-tests/testcases/hive_tpch/q10.result

```diff
@@ -1,4 +1,4 @@
--- delimiter: |; ignoreOrder: false; types: BIGINT|LONGNVARCHAR|DOUBLE|DOUBLE|LONGNVARCHAR|LONGNVARCHAR|LONGNVARCHAR|LONGNVARCHAR
```

Comment:
I see the value of being consistent, but I was trying to make the smallest set of changes that would solve the problem.  The other tests don't use VARCHAR columns, and so can pass without this change.